### PR TITLE
{bio}[foss/2019a,intel/2019a] GROMACS v2019.2

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 4 out of 5 (2 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/1f52c4908e2fed0fc2b546f37f0243f7 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (2 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/fc3c354bdc61adb193a78ffd73d34c77 for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 2 out of 4 (2 easyconfigs in this PR)
nic427 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/783500e8d7501c759db8491001595cc6 for a full test report.

**Comment by smoors** [comment]
the foss version gives 1 failed test:
```
/tmp/smoors.2055653/build/GROMACS/2019.2/foss-2019a/gromacs-2019.2/src/gromacs/hardware/tests/hardwaretopology.cpp:88: Failure
Expected: (hwTop.supportLevel()) >= (gmx::HardwareTopology::SupportLevel::Basic), actual: 4-byte object <01-00 00-00> vs 4-byte object <02-00 00-00>
Cannot determine basic hardware topology from hwloc. GROMACS will still

work, but it might affect your performance for large nodes.
Please mail gmx-developers@gromacs.org so we can try to fix it.
[  FAILED  ] HardwareTopologyTest.HwlocExecute (49 ms)
```
with intel there are 2 failed tests:
```
30/39 Test #30: CorrelationsTest .................***Exception: SegFault  0.01 sec
33/39 Test #33: TrajectoryAnalysisUnitTests ......***Exception: SegFault  0.01 sec
```
@akesandgren what do you think of these failures?

**Comment by akesandgren** [comment]
I think I need to re-add hwloc as dep. Missed that when i checked the config step apparently...
as for the intel problem... no clue. I didn't see any problems building on skylake.

**Comment by akesandgren** [comment]
Which hwloc version did it pick up?

**Comment by smoors** [comment]
we have a system hwloc version `1.11.8` installed.

**Comment by akesandgren** [comment]
Not what i asked... what hwloc version did it pick up during configure?

**Comment by smoors** [comment]
`-- Found Hwloc: /apps/brussel/CO7/skylake/software/hwloc/1.11.11-GCCcore-8.2.0/lib/libhwloc.so (found suitable version "1.11.11", minimum required is "1.5")-`
Since hwloc is not in the deps list I figured it must have used the system hwloc, but apparently not...

**Comment by akesandgren** [comment]
The i have no idea. Our system hwloc is 1.11.5 and configure picks up the one from foss/2019a 1.11.11
So there is no reason for this to fail.

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 2 out of 3 (2 easyconfigs in this PR)
nic169 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2650 v4 @ 2.20GHz, Python 2.7.5
See https://gist.github.com/c00560182e6fdb26e44cd71036405cf1 for a full test report.

File: easybuild/easyconfigs/g/GROMACS/GROMACS-2019.2-foss-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+GROMACS is a versatile package to perform molecular dynamics,
+ i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles.
+
+This is a CPU only build, containing both MPI and threadMPI builds.
+"""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+toolchainopts = {'openmp': True, 'usempi': True}
+
+source_urls = ['ftp://ftp.gromacs.org/pub/gromacs/']
```
Comment (boegel) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-2019.2-foss-2019a.eb]: @akesandgren Please prefer `https://` over `ftp://`, cfr. #7948

File: easybuild/easyconfigs/g/GROMACS/GROMACS-2019.2-intel-2019a.eb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+GROMACS is a versatile package to perform molecular dynamics,
+ i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles.
+
+This is a CPU only build, containing both MPI and threadMPI builds.
+"""
+
+toolchain = {'name': 'intel', 'version': '2019a'}
+toolchainopts = {'openmp': True, 'usempi': True}
+
+source_urls = ['ftp://ftp.gromacs.org/pub/gromacs/']
```
Comment (boegel) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-2019.2-intel-2019a.eb]: @akesandgren Please prefer `https://` over `ftp://`, cfr. #7948

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
node3119.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/f96dd735c1aba3a5ea8a8157088016ba for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
node2066.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/80fe64ba526acdae5f3589381f98a3c7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
node2591.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/347b9d3c83365c271450c46904e02c8a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2424.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/b60d01d6e2bbc97f9688ccebcdcb659a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
node3110.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/d641b589f7353a807d4644006ddb1639 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
node3110.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/c2dd91a6a99ec73c9cae4885182698b8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2016.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e360e7edaffdf8c5531cae541a3c9f2d for a full test report.

**Comment by boegel** [comment]
@akesandgren Hmm, segfaults on Intel Skylake X?

**Comment by akesandgren** [comment]
Yes I think it does for me too, because I've only built the foss version on the skylakes.
Wasn't there an easyblock fix for this in the works?

**Comment by boegel** [comment]
@akesandgren What type of fix? Doesn't ring a bell for me...

**Comment by akesandgren** [comment]
Ah, got my head turned on the wrong way. That one was for non-fma systems...

**Comment by boegel** [comment]
@akesandgren Is it worth pursuing this one, or shall we close it now we have GROMACS 2020 (see #9670)?

**Comment by akesandgren** [comment]
Drop it i think.

**Pull request closed without merging**